wgpu-core/src/instance.rs

```diff
@@ -144,6 +144,14 @@ impl<B: GfxBackend> Adapter<B> {
             wgt::Features::TEXTURE_COMPRESSION_BC,
             adapter_features.contains(hal::Features::FORMAT_BC),
         );
+        features.set(
```

Comment:
This code know when to expose the features. Where is the code that enables them for device creation, if requested?